omnigibson/examples/robots/curobo_example.py

```diff
@@ -0,0 +1,186 @@
+import math
+import os
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+import torch as th
+
+import omnigibson as og
+import omnigibson.lazy as lazy
+import omnigibson.utils.transform_utils as T
+from omnigibson.action_primitives.curobo import CuroboEmbodimentSelection, CuRoboMotionGenerator
+from omnigibson.macros import gm, macros
+from omnigibson.object_states import Touching
+from omnigibson.robots.holonomic_base_robot import HolonomicBaseRobot
+
+
+def plan_trajectory(cmg, target_pos, target_quat, emb_sel=CuroboEmbodimentSelection.DEFAULT):
+    # Generate collision-free trajectories to the sampled eef poses (including self-collisions)
+    successes, traj_paths = cmg.compute_trajectories(
+        target_pos=target_pos,
+        target_quat=target_quat,
+        is_local=False,
+        max_attempts=1,
+        timeout=60.0,
+        ik_fail_return=5,
+        enable_finetune_trajopt=True,
+        finetune_attempts=1,
+        return_full_result=False,
+        success_ratio=1.0,
+        attached_obj=None,
+        emb_sel=emb_sel,
+    )
+    return successes, traj_paths
+
+
+def test_curobo():
+    # Create env
+    cfg = {
+        "env": {
+            "action_frequency": 30,
+            "physics_frequency": 300,
+        },
+        "scene": {
+            "type": "Scene",
+        },
+        "objects": [
+            {
+                "type": "PrimitiveObject",
+                "name": "eef_marker_0",
+                "primitive_type": "Sphere",
+                "radius": 0.05,
+                "visual_only": True,
+                "position": [0, 0, 0],
+                "orientation": [0, 0, 0, 1],
+                "rgba": [1, 0, 0, 1],
+            },
+            {
+                "type": "PrimitiveObject",
+                "name": "eef_marker_1",
+                "primitive_type": "Sphere",
+                "radius": 0.05,
+                "visual_only": True,
+                "position": [0, 0, 0],
+                "orientation": [0, 0, 0, 1],
+                "rgba": [0, 1, 0, 1],
+            },
+        ],
+        "robots": [
+            {
+                "type": "R1",
+                "obs_modalities": "rgb",
+                "position": [0, 0, 0],
+                "orientation": [0, 0, 0, 1],
+                "self_collisions": True,
+                "action_normalize": False,
+                "rigid_trunk": False,
+                "controller_config": {
+                    "base": {
+                        "name": "JointController",
+                        "motor_type": "position",
+                        "command_input_limits": None,
+                        "use_delta_commands": False,
+                        "use_impedances": True,
+                        "kp": macros.robots.holonomic_base_robot.BASE_JOINT_CONTROLLER_POSITION_KP,
```

Comment:
@cremebrule I want to set the default kp for `JointController` of the base controller of all `HolonomicBaseRobot`s. What's the best way to achieve this? My current solution is pretty ugly.